.circleci/deselect_tests.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+import argparse
+import os.path
+from yaml import load as yaml_load
+from distutils.version import LooseVersion
+import sklearn
+from sklearn import __version__ as sklearn_version
+
+def evaluate_cond(cond, v):
+    if cond.startswith(">="):
+        return LooseVersion(v) >= LooseVersion(cond[2:])
+    if cond.startswith("<="):
+        return LooseVersion(v) <= LooseVersion(cond[2:])
+    if cond.startswith("=="):
+        return LooseVersion(v) == LooseVersion(cond[2:])
+    if cond.startswith("!="):
+        return LooseVersion(v) != LooseVersion(cond[2:])
+    if cond.startswith("<"):
+        return LooseVersion(v) < LooseVersion(cond[2:])
+    if cond.startswith(">"):
+        return LooseVersion(v) > LooseVersion(cond[2:])
+    return False
```

Comment:
Shouldn't the last 2 compare against cond[**_1_**:]?

I am slightly concerned about not getting any meaningful message if using a wrong syntax. But it's probably ok to not over-engineer this.

Reply:
Yes, good catch! I'll fix it right away.

Reply:
Fixed in e9f51e1eff2a727515f6bfb9abb7bdc0dce5b8bd. Unclear why GH does not mark the code chunk associated with the comment as outdates.